genesyscloud/resource_genesyscloud_user.go

```diff
@@ -1256,6 +1319,8 @@ func updateUserRoutingUtilization(d *schema.ResourceData, usersAPI *platformclie
 					return diag.Errorf("Failed to delete Routing Utilization for user %s: %s", d.Id(), err)
 				}
 			}
+
+			log.Printf("Updated user utilization")
```

Comment:
add in a id or user to this.
this log may not be of much use for backtracking and troubleshooting